initializers/vlans.yml

```diff
@@ -0,0 +1,9 @@
+#- name: vlan1
+#  site: AMS 1
+#  status: 1
```

Comment:
Would it be possible to use the actual word here, i.e. `Active`, `Reserved`, `Deprecated`?

See https://github.com/netbox-community/netbox-docker/blob/master/startup_scripts/110_devices.py#L52-L55 for how this was made for the `face` property in `devices.yml`.